langchain/schema.py

```diff
@@ -255,42 +255,16 @@ def clear(self) -> None:
         """Clear memory contents."""
 
 
-class MessageDB(ABC):
-    """Base interface for the buffer memory to be stored in a database."""
-
-    @abstractmethod
-    def __init__(self) -> None:
-        """Initialize the database."""
-
-    @abstractmethod
-    def read(
-        self, session_id: str
-    ) -> Union[str, List[BaseMessage], List[Dict[str, Any]]]:
-        """Retrieve history buffer from the database"""
-
-    @abstractmethod
-    def append(self, session_id: str, message: Union[HumanMessage, AIMessage]) -> None:
-        """Save conversation history to the database."""
-
-    @abstractmethod
-    def clear(self, session_id: str) -> None:
-        """Clear memory contents from the database for this session."""
-
-
-class MessageStore(ABC):
+class ChatMessageHistoryBase(ABC):
```

Comment:
A suggestion to name this `BaseChatMessageHistory`, following the pattern in existing classes, for example `BaseChatMemory`.

Reply:
Agreed, changing this now.